src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/AuthorizationController.cs

```diff
@@ -66,5 +67,19 @@ public IActionResult ApprenticeEmailRequired(long providerId)
             _logger.LogInformation($"Feature 'apprentice-email-required' is off for provider {providerId}");
             return NotFound();
         }
+
+        [HttpGet]
+        [Route("email-optional/{employerId:long}/{providerId:long}")]
```

Comment:
It doesn't really make sense to put these ids in the route. It believe it would read better as query string params

Reply:
I actually prefer it like this.. but I guess in this instance consumers could get confused which way around they go..